plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/pod_template.py

```diff
@@ -0,0 +1,36 @@
+from kubernetes.client import V1Container, V1EmptyDirVolumeSource, V1PodSpec, V1Volume, V1VolumeMount
```

Comment:
We need to add the `kubernetes` dependency to:

https://github.com/flyteorg/flytekit/blob/a7285067bfd90d53c373163c7db961fe09e3231d/plugins/flytekit-kf-pytorch/setup.py#L7